playwright/test/workPrototype.test.ts

```diff
@@ -0,0 +1,81 @@
+import {
+  workWithPhysicalAndDigitalLocation,
+  workWithRequestablePhysicalItem,
+  workWithPhysicalLocationOnly,
+  workWithDigitalLocationOnly,
+} from './contexts';
+import { makeDefaultToggleAndTestCookies } from './helpers/utils';
+import { baseUrl } from './helpers/urls';
+
+const domain = new URL(baseUrl).host;
+
+// Turn on the showPhysicalItems toggle
+beforeAll(async () => {
+  const defaultToggleAndTestCookies = await makeDefaultToggleAndTestCookies(
+    domain
+  );
+  const toggleOverrides = [
+    {
+      name: 'toggle_showPhysicalItems',
+      value: 'true',
+    },
+  ];
+  const overriddenCookies = defaultToggleAndTestCookies.map(cookie => {
+    const matchingOverrideCookie = toggleOverrides.find(
+      override => override.name === cookie.name
+    );
+    if (matchingOverrideCookie) {
+      return {
+        ...cookie,
+        value: matchingOverrideCookie.value,
+      };
+    } else {
+      return cookie;
+    }
+  });
+  await context.addCookies([
+    { name: 'WC_cookiesAccepted', value: 'true', domain: domain, path: '/' },
+    { name: 'WC_globalAlert', value: 'true', domain: domain, path: '/' },
+    ...overriddenCookies,
+  ]);
+});
+
+async function getWhereToFindItAndEncoreLink() {
+  const whereToFindIt = await page.$('h2:has-text("Where to find it")');
+  const encoreLink = await page.$('a:has-text("Request item")');
+
+  return {
+    whereToFindIt,
+    encoreLink,
+  };
+}
+
+describe(`Scenario 1: a user wants to see relevant information about where a work's items are located and the access status of those items where applicable`, () => {
```

Comment:
Are we working with UX/UR on these scenarios? Apologies if we are, when we started this it was always with the intention of having these tests universally agreed with as critical user flows.

Reply:
I assumed so, these were copied across from work.test.ts

Reply:
Ah, great, I didn't realise this - this is why I ask as we've always given ourselves loads of unwinding to do, could we not just start with a new "request items" suite?